src/App.js

```diff
@@ -1,7 +1,7 @@
 import React, { useState, useEffect } from "react";
 import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import Navbar from "./components/Navbar";
 // import "./App.css";
-// import { NavBar } from "./components/Navigation";
```

Comment:
Can Kerry's orginal nav bar component be kept be integrated in some way?

Reply:
I am thinking, after the other pull request, I check this and then merge my code in

Reply:
ah right 👍